src/richie/apps/core/templatetags/language_name.py

```diff
@@ -0,0 +1,23 @@
+"""Custom template tags related to FEATURES settings."""
```

Comment:
Do not forget to udpate the module doc string 😉

Reply:
Yes! A copy paste problem!